starfish/pipeline/filter/mean_high_pass.py

```diff
@@ -0,0 +1,94 @@
+import argparse
+from functools import partial
+from numbers import Number
+from typing import Callable, Union, Tuple, Optional
+
+import numpy as np
+from skimage import img_as_uint
+
+from starfish.errors import DataFormatWarning
+from starfish.image import ImageStack
+from scipy.ndimage.filters import uniform_filter
+from ._base import FilterAlgorithmBase
+
+
+class MeanHighPass(FilterAlgorithmBase):
+
+    def __init__(
+            self, size: Number, is_volume: bool=False, verbose: bool=False, **kwargs
+    ) -> None:
+        """Mean high pass filter
```

Comment:
I would add a block with the following to the documentation: 

```
See Also
--------
scipy.ndimage.filters.uniform_filter: 
https://docs.scipy.org/doc/scipy-0.19.0/reference/generated/scipy.ndimage.uniform_filter.html
```

Also as an imaging noob, I don't see the comparison between `MeanHighPass` <-> `UniformFilter` anywhere. Adding a few words of explanation here might be nice, or referencing the other filter in the docs to make the connection for a user. 

It could also be nice to add a little bit of explanation of what this filter is supposed to do or why to apply it.